util/src/main/java/org/hyperledger/besu/util/number/PositiveNumber.java

```diff
@@ -19,14 +19,18 @@
 
 import java.util.Objects;
 
-public class PositiveNumber {
+public class PositiveNumber extends Number {
```

Comment:
Is this essential to this PR?  Can it be spun out independently and done sooner?

Reply:
It is not essential to this PR

Reply:
Removed